app/models/ca_metadata_elements.php

```diff
@@ -1269,7 +1269,7 @@ static public function getElementDatatype($pm_element_code_or_id, array $options
 		$return_string = $options['returnAsString'] ?? false;
 
 		if(MemoryCache::contains($pm_element_code_or_id, 'ElementDataTypes')) {
-			$vm_return = (int)MemoryCache::fetch($pm_element_code_or_id, 'ElementDataTypes');
+			$vm_return = MemoryCache::fetch($pm_element_code_or_id, 'ElementDataTypes');
```

Comment:
@kehh We might want to note this, not sure if it will have knock on effects that might be difficult to track down.

Reply:
I hate to think of the knock on effects that lead you down this track in the first place